src/DocusignBundle.php

```diff
@@ -5,7 +5,7 @@
 namespace DocusignBundle;
 
 use DocusignBundle\DependencyInjection\Compiler\EnvelopeBuilderPass;
-use DocusignBundle\DependencyInjection\Compiler\PluginPass;
+
```

Comment:
you can remove this line, don't forget to run 
`vendor/bin/phpstan analyze`
and
`vendor/bin/php-cs-fixer fix`